agent/gen.sh

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p bash protoc-gen-go protoc-gen-go-grpc grpc-tools
+
+### Generate go code from protobuf
+set -euo pipefail
+
+rm -f *.pb.go
```

Comment:
why is this needed. it should be overwritten anyway ?

Reply:
👍